### PR TITLE
Feature/meridional convergence angle correction

### PR DESCRIPTION
**Pull request by rsasaki0109**
The meridian convergence angle is the angle of difference between true north and coordinate north.
The meridian convergence angle is calculated by the `getMeridianConvergence()` function.

input
- lat, lon, altitude
- xyz

output
- meridional convergence angle

image

test
```
./devel/lib/llh_converter/mcac_test
```

test result
```
-------------------------------------------------------------------------------------
Testing LatLon (35.1412, 35.1412) ... 
Calcalated Meridian Convergence Angle (-0.00177912)
TEST SUCCESS: |(-0.101936) - (-0.101925)| = 1.09271e-05< 0.0001
-------------------------------------------------------------------------------------
Testing LatLon (35.1888, 35.1888) ... 
Calcalated Meridian Convergence Angle (-0.00224893)
TEST SUCCESS: |(-0.128854) - (-0.128839)| = 1.52432e-05< 0.0001
```

### CODE REVIEW

**Comment by rsasaki0109** [comment]
@Tomoya-Sato I would like to ask you to review this one.

**Comment by rsasaki0109** [comment]
@Tomoya-Sato Maybe you forgot this PR, so I will ping you.

File: src/meridian_convergence_angle_correction.cpp [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+    offset_converted_vec.x = offset_xyz.x - xyz.x;
+    offset_converted_vec.y = offset_xyz.y - xyz.y;
+    xyz_by_offset_lla_converted_vec.x = xyz_by_offset_lla.x - xyz.x;
+    xyz_by_offset_lla_converted_vec.y = xyz_by_offset_lla.y - xyz.y;
+
+    double dot_norm = getDotNorm(offset_converted_vec, xyz_by_offset_lla_converted_vec);
+    double cross_norm = getCrossNorm(offset_converted_vec, xyz_by_offset_lla_converted_vec);
+
+    return atan2(cross_norm, dot_norm);
```
Comment (Tomoya-Sato) [inline src/meridian_convergence_angle_correction.cpp]: We should use `std::atan2` instead of `atan2`, due to compatibility.

Comment (rsasaki0109) [reply src/meridian_convergence_angle_correction.cpp]: Fixed.

**Pull request merged**